examples/resources/hpegl_vmaas_network/nsx_t_static_network.tf

```diff
@@ -1,20 +1,25 @@
 # (C) Copyright 2021 Hewlett Packard Enterprise Development LP
 
 resource "hpegl_vmaas_network" "test_net" {
-  name         = "tf_net_custom"
-  description  = "Network created using tf"
-  group_id     = data.hpegl_vmaas_group.tf_group.id
+  name         = "tf_nsx_t_static_network"
+  description  = "Static Network create using tf"
   scope_id     = data.hpegl_vmaas_transport_zone.tf_zone.provider_id
-  cidr         = "168.72.10.1/18"
+  cidr         = "193.2.3.1/20"
   gateway      = "168.72.10.1"
   primary_dns  = "8.8.8.8"
   scan_network = false
   pool_id = 7
   active       = true
-  config {
-    connected_gateway = data.hpegl_vmaas_router.tf_router.provider_id
-  }
```

Comment:
We need to check with Reuben, if we need to maintain compatibility here ? these may break existing automation, consumers if they are using it.